src/components/AppMap.vue

```diff
@@ -167,6 +167,7 @@
           </div>
           <div class="col">
             <b-btn size="sm" variant="danger" block @click="drawImport()"><i class="fas fa-file-import"></i> Import</b-btn>
+            <b-form-checkbox id="replace_checkbox" v-model="importReplace" name="replace_checkbox" value=true > Replace existing data</b-form-checkbox>
```

Comment:
```suggestion
            <b-form-checkbox v-model="importReplace">Replace existing data</b-form-checkbox>
```

I think you should be able to get rid of most of the stuff there. According to the docs b-form-checkbox defaults to setting the value to true (as a boolean) if the box is checked and false otherwise.